### ISSUE TITLE
Password reset without SMTP

### ISSUE THREAD

**Issue by Overlord23** [Ylianst/MeshCentral#1052]
Hi, 
I'm testing MeshCentral since a couple of days ago, nice work guys!
I'have a question, is it possible to reset an admin account password without configuring the SMTP client? My server is currently operating in LAN only without any access to the internet.
I took a look at meshcentral.db, is there any way to clear out the password from there or reset it through command line?

Cheers!

**Comment by Ylianst** [comment]
Good request. I will add a command line way to do it. Will be released later today.

**Comment by Ylianst** [comment]
Published MeshCentral v0.5.0-s with new commands to recover an administrative account.

```
node node_modules/meshcentral --help
node node_modules/meshcentral --listuserids
node node_modules/meshcentral --resetaccount [userid] --pass [password]
node node_modules/meshcentral --adminaccount [userid]
```

`ListUserIDs` will show you a list of all user identifiers.
`ResetAccount` will set a new password, unlock the account and disable all 2FA factors.
`AdminAccount` will elevate an account to site administrator.

Let me know if this works.

![MC2-AdminRecovery](https://user-images.githubusercontent.com/1319013/77261930-5efbd500-6c4f-11ea-93e8-5ffaafefc0e5.png)
